modules/utils.js

```diff
@@ -65,15 +65,8 @@ const findNode = (feature, searchId) => {
   });
 };
 
-const detectLastScenario = (node, searchId) => {
-  let isLastScenario = false;
-  node.children.forEach((child, index) => {
-    if (child.scenario) {
-      isLastScenario = child.scenario.id === searchId && index === node.children.length - 1;
-    }
-  });
-  return isLastScenario;
-};
+const detectLastScenario = (allScenarios, startedScenarios) =>
+  allScenarios.every((scenarioId) => startedScenarios.includes(scenarioId));
```

Comment:
Seems like now this function not detects the last scenario, but checks that all scenarios started.